build/templates/run-service-bus-test-worker.yml

```diff
@@ -0,0 +1,34 @@
+parameters:
```

Comment:
If we make it more generic in terms of naming, we might want to consider renaming this template as well

Reply:
Updated!